pkg/ddevapp/global_dotddev_assets/commands/host/phpmyadmin

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+## #ddev-generated: If you want to edit and own this file, remove this line.
+## Description: Prompt downloading phpMyAdmin if it is not installed.
+## Usage: phpmyadmin
+## Example: "ddev phpmyadmin"
+
+if [ "${DDEV_PROJECT_STATUS-running}" != "running" ] && [ -z "$no_recursion" ]; then
+  echo "Project ${DDEV_PROJECT} is not running, starting it"
+  ddev start
+  start_exit_code=$?
+  if [ $start_exit_code -ne 0 ]; then
+    exit $start_exit_code
+  fi
+  # run this script again, as the environment is updated after "ddev start"
+  no_recursion=true ddev "$(basename "$0")" "$@"
+  exit $?
+fi
```

Comment:
It is not necessary for the project to be in a running state. We can add DDEV_NONINTERACTIVE here

```suggestion
if [ "${DDEV_NONINTERACTIVE:-}" != "" ]]; then
  echo "Nothing has been changed."
  exit 0
fi
```

Reply:
I disagree, if I run a ddev command I always want ddev to start and run the command, or am I misunderstanding?

Reply:
Not always, I assume you just copied this condition from another global command, which requires a running project.
I tried your PR without this condition to be sure. and it worked.

Reply:
Ah thanks, I misunderstood your comment, I'll change it.